packages/paste-core/primitives/box/src/StyleFunctions.ts

```diff
@@ -69,6 +69,14 @@ export const PasteStyleProps = system({
   borderSpacing: true,
   tableLayout: true,
   fontVariantNumeric: true,
+  columnGap: {
+    property: 'columnGap',
+    scale: 'space',
+  },
+  rowGap: {
+    property: 'rowGap',
+    scale: 'space',
+  },
```

Comment:
Hmmm this is kinda wild, I wasn't expecting you to need to do this as it's part of the [default scale from Styled system](https://github.com/twilio-labs/paste/blob/main/packages/paste-libraries/styling/src/css-function/css.ts#L82), but they don't actually map it to a CSS property https://styled-system.com/api/.

That's annoying. 

Would love a jest test to prove the new Box prop works as expected

Reply:
Cool beans, just added one above ☝️ [quick link](https://github.com/twilio-labs/paste/pull/1743/files#diff-8ba175aa89f805d32092e9cdf1d8a22f7d0d9c81ef0dd1f478154bdb2bd6c99bR356)